lib/elixir/test/elixir/kernel_test.exs

```diff
@@ -856,11 +856,25 @@ defmodule KernelTest do
     end
   end
 
+  test "tl/2" do
+    assert tl([:one]) == []
+    assert tl([1, 2, 3]) == [2, 3]
+
+    assert_raise ArgumentError, "argument error", fn ->
+      tl([])
```

Comment:
Sorry folks, but this test is giving a warning,
> warning: this expression will fail with ArgumentError

Any idea how to bypass the warning?